### DISCUSSION TITLE
texttoolspy: My PyPi package

### DISCUSSION THREAD

**Discussion by MilesWK** [community/community#118593] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Show & Tell

### Body

# texttoolspy
**By MilesWK**

I am showcasing my first ever python package called [texttoolspy](https://pypi.org/project/texttoolspy/)!

What this does: It allows you to create simple and responsive "widgets" in the python project when running them in the terminal! 

Check it out on PyPi! (It has the GitHub Wiki link as well): [https://pypi.org/project/texttoolspy/](https://pypi.org/project/texttoolspy/)!

I hope you like it!

**Comment by ettaboyle** [comment]
Hi @MilesWK, we’re glad you’re here 😀

Thanks for sharing your (project/website/etc) with the community. Unfortunately, we currently do not allow self-promotion or advertising on the Community Discussions. We want to make sure there is space for users to ask questions without overwhelming them with other conversations. Thank you for helping us maintain a productive and tidy community for all our members.
